src/ActionsTests/ActionsTests.csproj

```diff
@@ -54,6 +54,9 @@
     <Reference Include="Axe.Windows.Desktop.Fakes" Condition="$(FAKES_SUPPORTED) == 1">
       <HintPath>..\Fakes.Prebuild\FakesAssemblies\Axe.Windows.Desktop.Fakes.dll</HintPath>
```

Comment:
Should you have removed the fakes references and the instances of the environment variable?

Reply:
No. ActionsTests still uses other fakes (this PR just removes the A11yElement fakes)

---
In reply to: [289189333](https://github.com/microsoft/axe-windows/pull/37#discussion_r289189333) [](ancestors = 289189333)